webpack.dev.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+
+module.exports = {
+    mode: 'development',
+    entry: path.resolve(__dirname, './src/index.js'),
+    module: {
+        rules: [
+            {
+                test: /\.(js|jsx)$/,
```

Comment:
all the duplication (similar config for dev and prod) should be moved to common webpack config file otherwise what's the point in common config?